core/types/transaction.go

```diff
@@ -263,6 +263,13 @@ func (tx *Transaction) To() *common.Address {
 	return &cpy
 }
 
+// Sender returns the sender of the transaction.
+func (tx *Transaction) Sender() common.Address {
+	signer := LatestSignerForChainID(tx.ChainId())
+	sender, _ := Sender(signer, tx)
```

Comment:
why are we not catching the error ?
Or is it not relevant for `Sender()`?

Reply:
Good question. The place I am using it (and most places it would likely be used) do not care about the error, because the transaction has already been sanitized. _However_, I should not ignore the error in a method like this, because I can't know that all future uses of this method are safe under the same assumption.

I will revert this commit and use `types.Sender()` to allow each usage to explicitly handle errors on a case-by-case basis

Reply:
Ok, I removed `tx.Sender()` and use `types.Sender()` as before